api/src/main/java/org/apache/cloudstack/api/command/admin/offering/CreateServiceOfferingCmd.java

```diff
@@ -56,7 +56,7 @@ public class CreateServiceOfferingCmd extends BaseCmd {
     @Parameter(name = ApiConstants.CPU_NUMBER, type = CommandType.INTEGER, required = false, description = "the CPU number of the service offering")
     private Integer cpuNumber;
 
-    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz.")
+    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz. In the case cgroups are used on KVM, this is interpreted to calculate the maximum number of shares.")
```

Comment:
according to https://libvirt.org/formatdomain.html#cpu-tuning

`
The optional shares element specifies the proportional weighted share for the domain. If this is omitted, it defaults to the OS provided defaults. NB, There is no unit for the value, it's a relative measure based on the setting of other VM, e.g. A VM configured with value 2048 will get twice as much CPU time as a VM configured with value 1024. The value should be in range [2, 262144] using cgroups v1, [1, 10000] using cgroups v2. Since 0.9.0
`
I am not sure if "the maximum number of shares" is correct.
maybe we could also add the link above in the description

Reply:
How about this @weizhouapache : 
```suggestion
    @Parameter(name = ApiConstants.CPU_SPEED, type = CommandType.INTEGER, required = false, description = "the CPU speed of the service offering in MHz. In the case cgroups are used on KVM, this is interpreted to calculate the number of shares. see also https://libvirt.org/formatdomain.html#cpu-tuning")
```

Reply:
that's fine !
@DaanHoogland